.github/workflows/docker_publish.yml

```diff
@@ -253,88 +77,42 @@ jobs:
         moab_versions : [
          5.3.0,
         ]
-    container: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-${{ matrix.compiler}}-ext-hdf5_${{ matrix.hdf5_versions}}-moab_${{ matrix.moab_versions }}:ci_testing
-
-    name: Installing and Testing DAGMC
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-        with:
-          submodules: recursive
-
-      - name: Setup
-        run: |
-          git config --global --add safe.directory ${GITHUB_WORKSPACE}
-          git config --global --add safe.directory /github/home/
-          git config --global --add safe.directory /github/home/.cache
-          git config --global --add safe.directory /github/home/.cache/pip
-          git config --global --add safe.directory /github/home/.cache/pip/http
-          echo "MOAB_VERSION=${{ matrix.moab_versions }}" >> $GITHUB_ENV
-          echo "COMPILER=${{ matrix.compiler }}" >> $GITHUB_ENV
-          echo "HDF5_VERSION=${{ matrix.hdf5_versions }}" >> $GITHUB_ENV
-          echo "REPO_SLUG=${GITHUB_REPOSITORY}" >> $GITHUB_ENV
-          echo "PULL_REQUEST=$(echo  $GITHUB_REF | cut -d"/" -f3)" >> $GITHUB_ENV
-          echo "DOUBLE_DOWN="OFF"" >> $GITHUB_ENV
-          ln -s $GITHUB_WORKSPACE /root/build_dir/DAGMC
-
-      - name: Install DAGMC
-        run: |
-          cd $GITHUB_WORKSPACE
-          /bin/bash "CI/scripts/install.sh"
-
-      - name: Run DAGMC Test
-        run: |
-          cd $GITHUB_WORKSPACE
-          /bin/bash "CI/scripts/tests.sh"
-
-  pushing_housekeeping_stable_img:
-    if: ${{ github.repository_owner == 'svalinn' }}
-    needs: [test_img]
-    runs-on: ubuntu-latest
-    env:
-      hdf5_versions: 1.10.4
-      hdf5_build_dir: hdf5_build_dir
-    strategy:
-      matrix:
-        ubuntu_versions : [18.04,]
-
-    name: Housekeeping':' latest -> stable
-    steps:
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
 
+    name: Installing DAGMC
+    steps:        
       - name: Log in to the Container registry
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         with:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Push Image as stable img
-        uses: akhilerm/tag-push-action@v1.0.0
-        with:
-          src: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-housekeeping:ci_testing
-          dst: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-housekeeping:stable
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
Where does this checkout go?  We are running in the image we have built that uses `/root/build_dir` as the place where it builds everything. The dockerfile creates `/root/build_dir/dagmc/build` and then invokes `cmake` with directory `../${dagmc}` but I can't tell where `dagmc` is set??  So I can't tell where it is invoking `cmake`?

Reply:
I'm thoroughly confused about how the dockerfile is even successful at building dagmc?!?! We never clone the repo from within the docker... what am I missing???

Reply:
I was missing something - it was clearly written in the comment!!!  The whole repo gets `COPY`ed into the docker image to the location in question, so it is now clear why/how this works. Sorry for the panic.